srl/src/main/java/edu/illinois/cs/cogcomp/srl/caches/SentenceDBHandler.java

```diff
@@ -0,0 +1,292 @@
+/**
```

Comment:
Apparently this is very similar to `TextAnnotationDBHandler.java`. Will try to keep only one of them.